README.md

```diff
@@ -0,0 +1,121 @@
+### tme-quant
+
+Curvelet-based image analysis utilities and a napari plugin, organized with a modern Python src/ layout.
```

Comment:
Let's reframe this along the lines of "Python translation of [CurveAlign](https://github.com/uw-loci/curvelets) with the goal of unifying cell and collagen analysis"